src/LocalizationWrapper.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component, PropTypes} from 'react';
+import defaultLocalizer from './util/localize-formatter';
```

Comment:
this is called `defaultLocalize` in the other file, we should name it the same consistently through the repo